eng/common/scripts/Prepare-Release.ps1

```diff
@@ -184,6 +184,7 @@ else
   exit 1
 }
 
+Sanitize-ChangeLogEntry -ChangeLogLocation $packageProperties.ChangeLogPath -VersionString $newVersion
```

Comment:
I would combine these into one function so we don't have to parse the changelog multiple times.

Reply:
We don't want to parse the raw change log file twice back to back here. We should wrap these checks in a common function or perhaps even add the "-SantizeEntry" parameter to the Confirm-ChangeLogEntry like you look to have started doing.